tests/rpc.rs

```diff
@@ -1,40 +1,18 @@
-use beerus::{
-    config::MAINNET_STARKNET_CHAINID,
-    gen::{
-        Address, BlockHash, BlockId, BlockNumber, BlockTag,
-        BroadcastedInvokeTxn, BroadcastedTxn, Felt, FunctionCall,
-        GetBlockWithTxHashesResult, GetBlockWithTxsResult, GetClassAtResult,
-        GetClassResult, GetTransactionByBlockIdAndIndexIndex, InvokeTxn,
-        InvokeTxnV1, InvokeTxnV1Version, PriceUnit, Rpc, StorageKey,
-        SyncingResult, Txn, TxnExecutionStatus, TxnHash, TxnReceipt,
-        TxnReceiptWithBlockInfo, TxnStatus,
-    },
+use beerus::gen::{
+    Address, BlockHash, BlockId, BlockNumber, BlockTag, BroadcastedInvokeTxn,
+    BroadcastedTxn, Felt, FunctionCall, GetBlockWithTxHashesResult,
+    GetBlockWithTxsResult, GetClassAtResult, GetClassResult,
+    GetTransactionByBlockIdAndIndexIndex, InvokeTxn, InvokeTxnV1,
+    InvokeTxnV1Version, PriceUnit, Rpc, StorageKey, SyncingResult, Txn,
+    TxnExecutionStatus, TxnHash, TxnReceipt, TxnReceiptWithBlockInfo,
+    TxnStatus,
 };
 
 mod common;
+mod starknet;
 
 use common::err::Error;
 
-#[tokio::test]
-#[allow(non_snake_case)]
-async fn test_specVersion() -> Result<(), Error> {
-    let ctx = setup!();
-
-    let ret = ctx.client.specVersion().await?;
-    assert_eq!(ret, "0.7.1");
-    Ok(())
-}
-
-#[tokio::test]
-#[allow(non_snake_case)]
-async fn test_chainId() -> Result<(), Error> {
```

Comment:
Why remove this test case?

Reply:
I guess formatting issue, same as one above. Returning it